i18n/en.json

```diff
@@ -0,0 +1,13 @@
+{
+	"@metadata": {
+		"authors": [
+			"Universal Omega"
+		]
+	},
+	"removepii-desc": "Creates the special page, Special:GenerateRandomHash.",
```

Comment:
Why not say what the special page does instead?

Reply:
Because this is just the first phase and I made that just say what it does so far, so eventually it will be expanded to include more stuff, another special page. Etc... it gets kind of complicated to try to explain that all here.